src/components/Navbar.css

```diff
@@ -0,0 +1,24 @@
+.Navbar {
+  position: absolute;
+  width: 100%;
+  background-color: #CAD959;
+  padding: 46px 282px;
+}
+
+a:link {
+  color: #34401A;
+  text-decoration: none;
+  font-size: 24px;
+  padding-left: 36px;
+  padding-right: 36px;
```

Comment:
Looks like you're using your design system. Great!
If you want to use .css files, I suggest looking into CSS variables.
Will make your life easier, and improve your code quality.